src/PrivateDns/PrivateDns/help/Get-AzPrivateDnsZone.md

```diff
@@ -65,7 +65,6 @@ Etag                  : xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
 Tags                  :
 NumberOfRecordSets    : 1
 MaxNumberOfRecordSets : 5000
```

Comment:
The output here does not match the actual output of Private DNS zone. Private DNS zone contains properties which displays counts of virtual network links as well. Did you copy-paste Public DNS here? Needs fixing.

Reply:
added them